Sources/Hypertext.swift

```diff
@@ -6,18 +6,23 @@
 //  Copyright © 2016 Sahand Nayebaziz. All rights reserved.
 //
 
-public protocol Renderable {
-    func render() -> String
-    func render(startingWithSpacesCount: Int) -> String
```

Comment:
I am glad you see 1.i and 1.ii. Thank you for reading and listening. I think I agree with the rest of your philosophy. I am beginning to lose the bigger picture over the past couple changes, so I'm going to load this file up locally and play with it and I will write back :)